keps/sig-network/20190125-ingress-api-group.md

```diff
@@ -233,9 +234,16 @@ Note: default value are permitted between API versions
 
 [api-conv-versions]: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#defaulting
 
+##### Defaults
+
+For a smooth upgrade process, the `PathType` field will be added to the `v1beta1`
+Ingress API. This will be an optional field in `v1beta1` defaulting to a value
+of `ImplementationSpecific`. In `v1`, the field will be required with no default
```

Comment:
... will be a required field (i.e. there is no default value)...

Reply:
Updated, thanks!